Project.toml

```diff
@@ -8,12 +8,11 @@ LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 MathOptInterface = "b8f27783-ece8-5eb3-8dc8-9495eed66fee"
 PrecompileTools = "aea7be01-6a6a-4083-8856-8a6e6704d82a"
 TOML = "fa267f1f-6049-4f14-aa54-33bafae1ed76"
-#Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [compat]
 MathOptInterface = "1"
 PrecompileTools = "1"
-julia = "1.8"
+julia = "1.6"
```

Comment:
Why'd you bump the minimum Julia version to 1.8? The current long-term support is Julia v1.6, so unless you're using some new features, things should just work.

Reply:
[Problem-Algorithm-Solver pattern](https://discourse.julialang.org/t/function-depending-on-the-global-variable-inside-module/64322/10) mention that 1.7 is needed, and I code the project in 1.8,  the CI checks on 1.8 and 1.9

Reply:
I changed the code to test on 1.6.

Reply:
1.6 :

 ```
ERROR: LoadError: LoadError: syntax: invalid assignment location "; V, A, G, b, g, d, u, N, M, J" around /home/runner/work/StatusSwitchingQP.jl/StatusSwitchingQP.jl/src/types.jl:213
```